src/settings/ubuntuSettings.json

```diff
@@ -132,7 +140,7 @@
             "fileUris": "[variables('ubuntuScripts')]"
           },
           "protectedSettings": {
-            "commandToExecute": "[concat('bash elasticsearch-install.sh -y', variables('commonShortOpts'), variables('commonInstallParams'))]"
+            "commandToExecute": "[concat('bash elasticsearch-install.sh -y', variables('commonShortOpts'), variables('commonInstallParams'), '; bash nginx-install.sh', variables('nginxInstallParams'))]"
```

Comment:
Nice touch so you don't have to call your nginx install script inside elasticsearch-install and deal with all the params :)

Reply:
This works in bash, and hope this works from ARM templates :) Need to test